packages/shared/src/components/sidebar/common.tsx

```diff
@@ -92,50 +96,86 @@ export const RawNavItem = classed(
 );
 
 export const ListIcon = ({ Icon }: ListIconProps): ReactElement => (
-  <Icon className="w-5 h-5" />
+  <Icon className="w-5 h-5 pointer-events-none" />
 );
-export const ItemInner = ({
-  item,
-  sidebarExpanded,
-}: ItemInnerProps): ReactElement => (
-  <>
+
+const ItemInnerIcon = ({ alert, icon }: SidebarMenuItem) => {
+  return (
     <span className="relative mr-3">
-      {item.alert}
-      {item.icon}
+      {alert}
+      {icon}
     </span>
+  );
+};
+
+const ItemInnerIconTooltip = ({
+  alert,
+  icon,
+  title,
+  tooltip = {},
+}: SidebarMenuItem) => (
+  <SimpleTooltip {...tooltip} content={title} placement="right">
     <span
       className={classNames(
-        'flex-1 text-left transition-opacity',
-        sidebarExpanded ? 'opacity-100 delay-150' : 'opacity-0',
+        'relative mr-3',
+        tooltip.visible !== undefined && 'pointer-events-none',
```

Comment:
Just noticed this does break the mobile view.
The one for the invitePeople seems to work.

<img width="432" alt="Screenshot 2021-12-23 at 10 18 28" src="https://user-images.githubusercontent.com/554874/147197370-3ca462ee-890f-4824-812c-784bcae48836.png">

Reply:
Oh, let me check this one.

Reply:
How did you check this one? It seems to work on the current deployment.

![Screen Shot 2021-12-23 at 2 29 55 PM](https://user-images.githubusercontent.com/13744167/147198280-c04f57f9-e85b-46ce-8401-4ebf21e5d67c.png)

Reply:
Oh this might be a edge-case issue that was already existing.

Steps to reproduce:
1. Open desktop version
2. Set to small version (non-expanded icon only)
3. Resize to mobile version
4. Open sidebar

Reply:
I think this is a different issue, this happens to me as well. Whenever I try to resize the browser manually, everything doesn't seem to follow all the time.

Reply:
I think this is a pre-existing issue, I suggest we fix it in a separate PR so we can test out the tooltips in preview.